src/averageAges.js

```diff
@@ -20,6 +20,30 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let summaryAge = 0;
+  let result = 0;
+
+  if (!century) {
+    const onlyMen = people.filter(person => person.sex === 'm');
+
+    summaryAge = onlyMen.reduce((prev, current) => {
+      return prev + current.died - current.born;
+    }, 0);
```

Comment:
avoid return where it is possible 
```suggestion
    summaryAge = onlyMen.reduce((prev, current) => (
      prev + current.died - current.born;
    ), 0);
```